src/content/blog/mariadb-dump-breaking-change.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "MariaDB Dump (mysqldump) Breaking Change"
+pubDate: 2024-06-14
+#modifiedDate: 2024-04-23
+summary: What is the breaking change in Mariadb-dump (mysqldump) and what does it mean for my DDEV projects?
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/06/weeping-sea-lion.png
+  alt: "Dall-E: an image of a seal posed and crying with big tear drops, something like the MariaDB logo"
+  credit: 'Dall-E: an image of a seal posed and crying with big tear drops, something like the MariaDB logo'
+categories:
+  - DevOps
+---
+
+## What happened with MariaDB and MysqlDump?
```

Comment:
```suggestion
## What happened with MariaDB and MySQL Dump?
```
since later in the article it is referred to mariadb-dump and mysqldump it is probably better it use the correct word mark for MySQL since isnt referring to mariadb and mysqldump but to mariadb dump and mysql dump?